connect/cli/plugins/project/report/validations.py

```diff
@@ -0,0 +1,146 @@
+
+#  Copyright © 2022 CloudBlue. All rights reserved.
+
+import json
+import os
+import sys
+import inspect
+import importlib
+
+from connect.reports.parser import parse
+from connect.reports.validator import (
+    validate,
+    validate_with_schema,
+)
+from connect.cli.plugins.project.validators import (
+    get_code_context,
+    load_project_toml_file,
+    ValidationItem,
+    ValidationResult,
+)
+
+
+def validate_pyproject_toml(project_dir, context):
+    messages = []
+    data = load_project_toml_file(project_dir)
+    if isinstance(data, ValidationResult):
+        return data
+
+    descriptor_file = os.path.join(project_dir, 'pyproject.toml')
+    dependencies = data.get('tool', {}).get('poetry', {}).get('dev-dependencies', {})
+
+    if not dependencies or 'connect-cli' not in dependencies:
+        messages.append(
+            ValidationItem(
+                'WARNING',
+                'No dependency on *connect-cli* has been found.',
```

Comment:
Better `No development dependency....`

Reply:
Probably we can add a validation that the version of the CLI is the latest.
The CLI already have a version check.
Look at it please.